README.md

```diff
@@ -55,16 +55,16 @@ Let me know! Open an issue (or PR) or write in [slack](https://elmlang.slack.com
 
 ### Setup
 
-```elm
-elm package install
+```shell
+elm-package install
```

Comment:
I just wanted it to be consistent, but actually the ones without "-" are easier to type (and read). I update the readme to use them everywhere 😊